src/client/src/Stream.c

```diff
@@ -1570,6 +1582,7 @@ STATUS getStreamMetrics(PKinesisVideoStream pKinesisVideoStream, PStreamMetrics
 
     // Store the frame rate and the transfer rate
     pStreamMetrics->currentFrameRate = pKinesisVideoStream->diagnostics.currentFrameRate;
+    pStreamMetrics->elementaryFrameRate = pKinesisVideoStream->diagnostics.elementaryFrameRate;
```

Comment:
You cannot do this.  This field is only available in V2 of the struct, see:  https://github.com/awslabs/amazon-kinesis-video-streams-pic/blob/master/src/client/src/Stream.c#L1578

You need to add a new section at the end of it first like this to bail out early for V1 of the struct

Reply:
Good catch! Fixed for both client and stream